projects/Mallard/src/components/article/article-header/long-read-header.tsx

```diff
@@ -19,34 +45,46 @@ const LongReadHeader = ({
     kicker,
     standfirst,
 }: ArticleHeaderProps) => {
+    const { height } = useDimensions()
+    const isTablet = useMediaQuery(width => width >= Breakpoints.tabletVertical)
```

Comment:
Not quite sure how many times we'll use this so maybe it's fine here, but we could probably export helpers like `tablet` for that lambda 🤷‍♂

Reply:
oooh that would rule yeah, just an object with some cases?